apps/crn-frontend/src/network/interest-groups/state.ts

```diff
@@ -48,18 +48,12 @@ export const interestGroupsState = selectorFamily<
         newInterestGroups === undefined ||
         newInterestGroups instanceof DefaultValue
       ) {
-        const oldInterestGroups = get(interestGroupIndexState(options));
-        if (!(oldInterestGroups instanceof Error)) {
-          oldInterestGroups?.ids?.forEach((id) =>
-            reset(interestGroupState(id)),
-          );
-        }
```

Comment:
This code had no test coverage, nobody knew what it was for, and never seems to be called at runtime, so I removed it.